design-library/source/_patterns/02-molecules/03-section/_section-list.scss

```diff
@@ -0,0 +1,4 @@
+.post-list {
```

Comment:
Could you rename to `_section.scss`?

Reply:
Actually, it looks like we need a new pattern called section_list. Can you add a new pattern and directory and move the `section_list` files into it?